client/src/components/Charts/DayCounter.jsx

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import styled from 'styled-components';
+
+dayjs.extend(relativeTime);
+
+const URL = process.env.REACT_APP_URL;
+const USER_ID = 9999;
+
+const DayCounter = () => {
+  const [lastSessionDate, setLastSessionDate] = useState();
+  const [dateDifference, setDateDifference] = useState();
```

Comment:
Do you mean with `dateDifference` time period? I think the name is harder to understand. 😅

Reply:
I have done some renaming for improved clarity.